libcloud/common/openstack_identity.py

```diff
@@ -571,14 +571,13 @@ class OpenStackIdentityConnection(ConnectionUserAndKey):
     timeout = None
 
     def __init__(self, auth_url, user_id, key, tenant_name=None,
+                 domain_name=None, token_scope=None,
                  timeout=None, parent_conn=None):
         super(OpenStackIdentityConnection, self).__init__(user_id=user_id,
                                                           key=key,
                                                           url=auth_url,
                                                           timeout=timeout)
 
-        self.auth_url = auth_url
```

Comment:
(just thinking out loud)

I assume you removed those attributes since they are duplicated aka are declared again a couple of lines below, right?